mulog-core/test/com/brunobonacci/mulog/jvm_metrics_test.clj

```diff
@@ -0,0 +1,29 @@
+(ns com.brunobonacci.mulog.jvm-metrics-test
+  (:require [com.brunobonacci.mulog.jvm-metrics :refer [jvm-sample]]
+            [midje.sweet :refer [facts fact => just anything]]
+            [clojure.spec.test.alpha :as st])
+  (:import (java.lang.management ManagementFactory)))
+
+(st/instrument 'com.brunobonacci.mulog.jvm-metrics/capture-memory)
+(st/instrument 'com.brunobonacci.mulog.jvm-metrics/capture-memory-pools)
+
+(fact "it should capture JVM metrics for some key groups"
+  (jvm-sample {:memory {:heap true
+                        :buffers true}
+               :gc {:collections true
+                    :duration true}})
+  => {:memory {:used_heap 3487563
+               :max_heap 345364567}
+      :gc {:collections 45
+           :duration 345323}})
```

Comment:
this test is likely to fail at random as the used memory depends on the environment and previous execution
you should probably have a look at [Midje checkers for collections](https://github.com/marick/Midje/wiki/Checkers-for-collections-and-strings)
and use something like:

``` Clojure
(contains {:used_heap int?  :max_heap int?}) 
```

Reply:
I know it's not a good test, I just copy and pasted the example from the issue so I could look it up. I'll comment it and make a better test.